src/views/Rules/Rule/index.jsx

```diff
@@ -160,11 +160,11 @@ export default function Rule({ isNewRule, ...props }) {
       fallbackMapping: rule.fallbackMapping,
       headerArchitecture: rule.headerArchitecture,
       instructionSet: rule.instructionSet,
-      jaws: rule.jaws,
+      jaws: rule.jaws === '-' ? null : rule.jaws === 'true',
```

Comment:
Instead of hard-coding `-`, we can use the already defined constant `EMPTY_MENU_ITEM_CHAR`. Same comment applies for the others.